docs/source/ProgrammingGuide/DeprecatedInitialization.md

```diff
@@ -0,0 +1,117 @@
+
+During initialization one or more execution spaces will be initialized and assigned to one of the following aliases.
+
+```c++
+Kokkos::DefaultExecutionSpace;
+```
+```c++
+Kokkos::DefaultHostExecutionSpace;
+```
+
+`DefaultExecutionSpace` is the execution space used with policies and views where one is not explicitly specified.  Primarily, Kokkos will initialize one of the heterogeneous backends (CUDA, OpenMPTarget, HIP, SYCL) as the `DefaultExecutionSpace` if enabled in the build configuration.  In addition, Kokkos requires a `DefaultHostExecutionSpace`.  The `DefaultHostExecutionSpace` is default execution space used when host operations are required.  If one of the parallel host execution spaces are enabled in the build environment then `Kokkos::Serial` is only initialized if it is explicitly enabled in the build configuration.  If a parallel host execution space is not enabled in the build configuration, then `Kokkos::Serial` is initialized as the `DefaultHostExecutionSpace`.
+Kokkos chooses the two spaces using the following list:
+
+1. `Kokkos::Cuda`
+2. `Kokkos::Experimental::OpenMPTarget`
+3. `Kokkos::Experimental::HIP`
+4. `Kokkos::Experimental::SYCL`
+5. `Kokkos::OpenMP`
+6. `Kokkos::Threads`
+7. `Kokkos::Experimental::HPX`
+8. `Kokkos::Serial`
+
+The highest execution space in the list which is enabled is Kokkos' default execution space, and the highest enabled host execution space is Kokkos' default host execution space. For example, if  `Kokkos::Cuda`, `Kokkos::OpenMP`, and `Kokkos::Serial` are enabled, then `Kokkos::Cuda` is the default execution space and `Kokkos::OpenMP` is the default host execution space.<sup>1</sup>  In cases where the highest enabled backend is a host parallel execution space the `DefaultExecutionSpace` and the `DefaultHostExecutionSpace` will be the same.
+
+Command-line arguments come in "prefixed" and "non-prefixed" versions. Prefixed versions start with the string `--kokkos-`. [`Kokkos::initialize`](../API/core/initialize_finalize/initialize) will remove prefixed options from the input list, but will preserve non-prefixed options. Argument options are given with an equals (`=`) sign. If the same argument occurs more than once, the last one is used. Furthermore, prefixed versions of the command line arguments take precedence over the non-prefixed ones. For example, the arguments
+
+    --kokkos-threads=4 --threads=2
+
+set the number of threads to 4, while
+
+    --kokkos-threads=4 --threads=2 --kokkos-threads=3
+
+set the number of threads to 3. Table 5.1 gives a full list of command-line options.
+
+<h4>Table 15.1: Command-line options for Kokkos::initialize</h4>
+
+Argument | Description
+:---      | :---
+--kokkos-help  <br/> --help   | print this message
+--kokkos-threads=INT <br/> --threads=INT  | specify total number of threads or number of threads per NUMA region if used in conjunction with `--numa` option.
```

Comment:
`kokkos-threads` vs. `kokkos-numa` is a bit unclear.  Do you use one or the other, or both at the same time in certain cases?  In any case, please crisply distinguish these options.